src/App/Components/TermsAndConditions/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import propTypes from 'prop-types';
+import Navbar from '../Common/NavProfile/navbar';
+import Footer from '../Common/Footer';
+import terms from '../../../Redux/Actions/terms';
+
+export class TermsAndConditions extends Component {
+  componentDidMount() {
+    const { getTerms } = this.props;
+    getTerms();
+  }
+
+  render() {
+    const { termsAndCondReducer: { termsAndConditions } } = this.props;
+    return (
+      <div className="">
+        <Navbar />
+        <div className="position-relative fixed-top navigation-info">
+          <span>Terms and Conditions</span>
+        </div>
+        <div className="container termsAndConditions ">
+          <div className="content">{termsAndConditions || 'Loading, please wait...'}</div>
+        </div>
+        <Footer />
+      </div>
+    );
+  }
+}
+TermsAndConditions.propTypes = {
+  getTerms: propTypes.func.isRequired,
+  termsAndCondReducer: propTypes.object
+};
+export const mapStateToProps = ({ termsAndCondReducer }) => ({ termsAndCondReducer });
+export const mapDispatchToProps = dispatch => ({ getTerms: () => dispatch(terms()) });
```

Comment:
I was comfortable with this, and I am willing to make it clear to whoever is not getting it since I am not familiar with the bindActionCreators.

Reply:
and Based to the research I made about the bindActionCreators I don't think I need them since I am using  Redux
https://redux.js.org/api/bindactioncreators#bindactioncreatorsactioncreators-dispatch